api/v1/tetragon/kernel.proto

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 Authors of Tetragon
```

Comment:
nit: Do we really need to create a new file for this `enum`? Why not just add this to `tetragon.proto`?

Reply:
You are right, made it part of tetragon.proto even if it's getting bigger, but there are external tools that won't expect a new .proto definition ;-)